scripts/template_openxr_dispatchstatic.hpp

```diff
@@ -50,6 +50,8 @@ namespace OPENXR_HPP_NAMESPACE {
  *
  * @ingroup dispatch
  */
+
+/*{protect_proto_begin()}*/
```

Comment:
same here - I don't think this needs a template call, it can just be the fixed thing defined in the OpenXR headers.